app/modules/ags/partials/_wellaged.html

```diff
@@ -29,7 +29,7 @@
                         <textarea class="form-control" rows="5" id="description" name="description" ng-model="selectedNode.model.attributes.description" ng-change="selectedNode.model.trigger('change:description')" "required></textarea>
                     </div>
 
-                    <div class="form-group" ng-if="undefined !== selectedNode.model.attributes.assumed" ng-class="{ 'has-error': agForm.title.$dirty && agForm.title.$invalid}">
+                    <div ng-show='selectedNode.model.get("type") == "wellaged.Statement"' class="form-group"ng-class="{ 'has-error': agForm.title.$dirty && agForm.title.$invalid}">
```

Comment:
Is there a reason to change ng-if to ng-show?
ng-show creates the dom element and controls the visibility where the ng-if appends the dom element  only if true. In general, you should prefer ng-if.